### PR TITLE
add style for table formatting to docs contribution

### PR DESCRIPTION
**Pull request by techdocsmith**
### Description

Clarifies the style for markdown tables in Apache Druid.

- [x] been self-reviewed.

### CODE REVIEW

**Review by 317brian — CHANGES_REQUESTED** [review]
The change itself LGTM. I think the spell check is failing on `IntelliJ`, so it should pass once the spelling file is updated

File: docs/development/docs-contribute.md [hunk 1]
```diff
@@ -169,6 +169,32 @@ HUMAN_READABLE_BINARY_BYTE_FORMAT(value[, precision])
 HUMAN_READABLE_BINARY_BYTE_FORMAT(value, \[precision])
 :::
 
+#### Markdown table format
+
+When editing or adding tables, do not include extra characters to "prettify" the table format within the markdown source.
```
Comment (317brian) [inline docs/development/docs-contribute.md:174]:
```suggestion
When editing or adding tables, do not include extra characters to "prettify" the table format within the Markdown source.
```

**Review by 317brian — APPROVED** [review]
LGTM, the rendered output looks good:

image

File: docs/development/docs-contribute.md [hunk 2]
```diff
@@ -169,6 +169,32 @@ HUMAN_READABLE_BINARY_BYTE_FORMAT(value[, precision])
 HUMAN_READABLE_BINARY_BYTE_FORMAT(value, \[precision])
 :::
 
+#### Markdown table format
+
+When editing or adding tables, do not include extra characters to "prettify" the table format within the Markdown source.
+Some code editors may format tables by default.
+For an example of how to disable it, see the [IntelliJ formatting file for Druid](https://github.com/apache/druid/blob/master/dev/druid_intellij_formatting.xml#L78).
```
Comment (abhishekrb19) [inline docs/development/docs-contribute.md:176]: Or it'd be better to import the code style file - https://github.com/apache/druid/blob/master/dev/style-conventions.md

**Review by abhishekrb19 — APPROVED** [review]
LGTM with one small suggestion

File: docs/development/docs-contribute.md (hunk 2 above)
Comment (techdocsmith) [inline docs/development/docs-contribute.md:176]:
```suggestion
See the [developer Style Guide for more information](https://github.com/apache/druid/blob/master/dev/style-conventions.md).
```
Thanks for the suggestion @abhishekrb19 . See revision.

File: docs/development/docs-contribute.md [hunk 3]
```diff
@@ -169,6 +169,32 @@ HUMAN_READABLE_BINARY_BYTE_FORMAT(value[, precision])
...
+
+When editing or adding tables, do not include extra characters to "prettify" the table format within the Markdown source.
+Some code editors may format tables by default.
+See the developer [style guide](https://github.com/apache/druid/blob/master/dev/style-conventions.md) for more information.
+
+:::tip
+
+**Correct**
+
+```
```
Comment (asdf2014) [inline docs/development/docs-contribute.md:182]:
```suggestion
```markdown
```

File: docs/development/docs-contribute.md [hunk 4]
```diff
@@ -169,6 +169,32 @@ HUMAN_READABLE_BINARY_BYTE_FORMAT(value[, precision])
...
+| Column 1 | Column 2 | Column 3 |
+| --- | --- | --- |
+| value 1 | val 2 | a-very-long-value 3 |
+```
+
+**Incorrect**
+
+```
```
Comment (asdf2014) [inline docs/development/docs-contribute.md:190]:
```suggestion
```markdown
```

**Review by asdf2014 — APPROVED** [review]
It would be better to add `markdown` to the code block :smile:

**Pull request merged**